testing/e2e/src/routes/api.middleware-test.ts

```
@@ -6,8 +6,36 @@ import {
+import type {
+  AttributeValue,
+  Attributes,
+  Context,
+  Histogram,
+  Meter,
+  MetricOptions,
+  Span,
+  SpanContext,
+  SpanStatus,
+  Tracer,
+} from '@opentelemetry/api'
```

ESLint

[error] `@opentelemetry/api` type import should occur after import of `zod`

(import/order)